compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/LeakDetector.kt

```diff
@@ -26,22 +26,17 @@ class LeakDetector {
         weakReferences.add(WeakReference(obj))
     }
 
-    fun noLeak(): Boolean {
+    fun isAnyWasGC(): Boolean {
```

Comment:
I think this now checks whether *any* observed object was collected. But what we want is to make sure that *all* observed objects were collected, no?

Reply:
Yes, but in my tests checking for all objects works much worse. Seems GC doesn't GC all windows. Not sure if that is a bug, or GC behavior. I lean towars the later.

Added a comment.